### DISCUSSION TITLE
Getting Started with 6x10 RGB MATRIX for XIAO | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1001] [Q&A]
# Getting Started with 6x10 RGB MATRIX for XIAO | Seeed Studio Wiki

Getting Started with 6x10 RGB MATRIX for XIAO

https://wiki.seeedstudio.com/rgb_matrix_for_xiao/

**Comment by giannisj5** [comment]
https://community.home-assistant.io/t/seeed-studio-6x10-rgb-matrix-on-xiao-esp32s3/629867

**Comment by MatthewJeffson** [comment]
Wow! Thank you so much for providing the information! This is very helpful and we will write a wiki about with it.
Thanks!
Matthew

**Comment by vluz** [comment]
Compiled 11 animations from various sources here: 

https://github.com/vluz/RGBMatrixAnim

**Reply by nfs0619** [reply]
Wow, thank you for sharing!

**Comment by bevis-hp** [comment]
"Meanwhile, all the example programs provided in this tutorial will work on other XIAOs as well" - This is untrue. This tutorial does not work with the Xiao MG24 as the neopixel library is not compatible with this microcontroller. It also seems that even the SiliconLabs ezWS2812.h library does not work either as this uses the SPI pin which is not D0 on the Xiao MG24 but D10 so the pins do not align. I have tried the ezWS2812gpio library to with no avail...

**Reply by Lukilyy** [reply]
Hi! At the moment, the officially supported platforms of the Adafruit_NeoPixel library do not explicitly include XIAO MG24 / EFR32MG24. Based on your findings, the issue may indeed be related to platform compatibility.

We will further verify the compatibility of the RGB Matrix examples on XIAO MG24 and improve the related Wiki documentation accordingly.

Thanks again for your testing and feedback.

**Comment by DevanMetz** [comment]
Hello, I just published a little bit-bang driver for using this with Seeed XIAO nRF54L15. Hope it helps. https://github.com/DevanMetz/6x10-RGB-WS2812-Matrix-for-XIAO-nRF54L15
